packages/ember-data/tests/integration/adapter/query-record-test.js

```diff
@@ -0,0 +1,81 @@
+var Person, store, env;
+var run = Ember.run;
+
+module("integration/adapter/query-record - Query one record with a query hash", {
```

Comment:
This is not an adapter integration test, as those are integration tests of adapters. I would put in a tests/integration/store or something like that